seedpod_ground_risk/ui_resources/coordbox.py

```diff
@@ -0,0 +1,91 @@
+import os
+from pathlib import Path
+
+from PySide2.QtCore import Property, Signal, Slot, Qt, QUrl
+from PySide2.QtPositioning import QGeoCoordinate
+from PySide2.QtQuickWidgets import QQuickWidget
+from PySide2.QtWidgets import (
+    QDialog,
+    QDialogButtonBox,
+    QDoubleSpinBox,
+    QHBoxLayout,
+    QLabel,
+    QToolButton,
+    QVBoxLayout,
+    QWidget,
+)
+
+CURRENT_DIRECTORY = Path(__file__).resolve().parent
+
+
+# TODO: Give the user feedback on where their coordinate is when clicked
+class MapDialog(QDialog):
+    def __init__(self, geo_widget):
+        super().__init__(geo_widget)
+        self.setWindowTitle("Map")
+        self.map_widget = QQuickWidget(resizeMode=QQuickWidget.SizeRootObjectToView)
+        self.map_widget.rootContext().setContextProperty("controller", geo_widget)
+        filename = os.fspath(CURRENT_DIRECTORY / "coord_map.qml")
+        url = QUrl.fromLocalFile(filename)
+        self.map_widget.setSource(url)
+
+        button_box = QDialogButtonBox()
+        button_box.setOrientation(Qt.Horizontal)
+        button_box.setStandardButtons(QDialogButtonBox.Cancel | QDialogButtonBox.Ok)
+
+        lay = QVBoxLayout(self)
+        lay.addWidget(self.map_widget)
+        lay.addWidget(button_box)
+
+        button_box.accepted.connect(self.accept)
+        button_box.rejected.connect(self.reject)
+
+
+class GeoWidget(QWidget):
+    coordinate_changed = Signal(name="coordinateChanged")
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self._coordinate = QGeoCoordinate(0, 0)
+
+        self._lat_spinbox = QDoubleSpinBox(
+            minimum=49.0, maximum=56.0  # , valueChanged=self.handle_value_changed,
```

Comment:
These are the bounds for England only? Please add comment explaining this. In future the area could be expanded and this would become a bug.

Reply:
Also remove commented code if it not going to be used.